openscad

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
Please add a .sh file extension.

Reply:
I mean just for the source file, of course rename it during install.

Reply:
done